framework/rendering/postprocessing_renderpass.cpp

```diff
@@ -429,7 +449,13 @@ void PostProcessingRenderPass::prepare_draw(CommandBuffer &command_buffer, Rende
 			if (const uint32_t *sampled_attachment = it.second.get_target_attachment())
 			{
 				auto *image_rt = it.second.get_render_target();
-				sampled_attachments.insert({image_rt, *sampled_attachment});
+				auto  packed_sampled_attachment = *sampled_attachment;
+
+				// pack sampled attachment
+				if (it.second.is_depth_resolve())
```

Comment:
(code-style) add braces even if single-line statement

Reply:
just checked the code-style guide and it does allow for this, approving https://google.github.io/styleguide/cppguide.html#Conditionals